### ISSUE TITLE
Flagged as "abandoned" by WordFence & WordPress.org

### ISSUE THREAD

**Issue by amas008** [aramk/crayon-syntax-highlighter#435]
This is a great plug-in, that I use on several sites. But, both WordFence (the WordPress security plug-in) and the WordPress.org site (https://wordpress.org/plugins/crayon-syntax-highlighter/) are now flagging this as "abandoned":

The Plugin "Syntax Highlighter for WordPress" appears to be abandoned (updated April 9, 2012, tested to WP 3.1.4).
Type: Plugin Abandoned

Is there any chance of pushing an update into the WordPress plugin system that contains the latest bug fixes and updated compatibility information? This will keep this "abandoned" message away for another 5 years or so, I hope!

Thanks for all your work on this. _Andrew_

**Comment by Abijeet** [comment]
Yes, came here to say that. I thought that this project would be deserted. Would it be possible to push an upgrade?

**Comment by yobyot** [comment]
I'd also appreciate a push to reset WordFence. I'm a big fan of this plugin. 

Thanks.

**Comment by aramk** [comment]
I'll look into this and push an update soon.

**Comment by abdulrahman19** [comment]
@aramk yes please!

**Comment by DBJDBJ** [comment]
@aramk I am very certain good WordFence people will listen to you if you contact them about this issue. WP plugin status is very crude "system" ... which also points the finger to "Automatic" people.  Sigh ... Actually such a "system" does not exist AFAIK. Having it will considerably improve the security of the WP. Second sigh. Ok, ok let me go "over there" and try to rock the boat. :)

**Comment by DesarrolladorAndroid** [comment]
@aramk kotlin support with this update?

**Comment by florianwalther-private** [comment]
Any update? I want to know if I have to switch to a different plugin or if I can keep Crayon? 
Also any chances to get Kotlin support?

**Comment by RichardCoffee** [comment]
I have [a fork](https://github.com/RichardCoffee/crayon-syntax-highlighter) that includes PR #414 in it.  It also includes some, not all, of the other outstanding PRs in it as well.

**Comment by florianwalther-private** [comment]
Any updates?

**Comment by florianwalther-private** [comment]
Did I understand it correctly that aramk was going to update it?
But this was 3.5 months ago
